core/core-workflow-steps/src/main/java/org/eclipse/sw360/antenna/workflow/generators/ArtifactForHTMLReport.java

```diff
@@ -55,6 +57,13 @@ public LicenseInformation getLicense() {
         return license;
     }
 
+    public String getLicenseAsHtmlListItem(License license) {
```

Comment:
this is a util function in an unrelated class. Maybe move it to a util class.

Reply:
A new util class or is `ArtifactUtils` suitable?

Reply:
it should be something here in in this package, since it is only for this special html report valid